src/main/java/bio/terra/service/dataset/flight/create/DatasetCreateFlight.java

```diff
@@ -55,7 +57,10 @@ public DatasetCreateFlight(FlightMap inputParameters, Object applicationContext)
         new AuthorizeBillingProfileUseStep(
             profileService, datasetRequest.getDefaultProfileId(), userReq));
 
-    // Get or create the project where the dataset resources will be created for GCP
+    // Get a new google project from RBS and store it in the working map
+    addStep(new CreateDatasetGetProjectStep(bufferService));
+
+    // Get or initialize the project where the dataset resources will be created
     addStep(new CreateDatasetGetOrCreateProjectStep(resourceService, datasetRequest));
```

Comment:
I wanted to rename `CreateDatasetGetOrCreateProjectStep` to `CreateDatasetInitializeProjectStep` since we're no longer creating the project ourselves (and also do this for the snapshot creation and file ingest flights). I think this can happen in a follow-on PR though since it's just a rename.